contract-address/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "contract-address"
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+license = "MIT"
+homepage = "https://github.com/paritytech/parity-common"
+repository = "https://github.com/paritytech/parity-common"
+description = "A utility crate to create an ethereum contract address"
+documentation = "https://docs.rs/contract-address/"
+edition = "2018"
+readme = "README.md"
+
+[dependencies]
+ethereum-types = { version = "0.7", path = "../ethereum-types" }
```

Comment:
I've tried to make it work with `default-features = false`, but failed :(
Any suggestions are welcome!